### PR TITLE
CHANGE: add es-index project to the billing aggregator

### PR DESCRIPTION
**Pull request by violetbrina**

### CODE REVIEW

File: cpg_infra/billing_aggregator/aggregate/seqr.py [hunk 1]
```diff
@@ -312,18 +312,20 @@ def migrate_entries_from_bq(
             service, sku, usage_start_time, usage_end_time, labels, system_labels,
             location, export_time, cost, currency, currency_conversion_rate, usage,
             credits, invoice, cost_type, adjustment_info
-        FROM `{GCP_BILLING_BQ_TABLE}`
+        FROM @billing_table
```
Comment (illusional) [inline cpg_infra/billing_aggregator/aggregate/seqr.py:315]: Now that this doesn't have placedholders, you could remove the `f` bit of the `f"string"`

File: cpg_infra/billing_aggregator/aggregate/seqr.py [hunk 2]
```diff
@@ -312,18 +312,20 @@ def migrate_entries_from_bq(
...
     """
 
+    projects = [utils.SEQR_PROJECT_ID, utils.ES_INDEX_PROJECT_ID]
     job_config = bq.QueryJobConfig(
         query_parameters=[
+            bq.ScalarQueryParameter('billing_table', 'STRING', GCP_BILLING_BQ_TABLE),
             bq.ScalarQueryParameter('start', 'STRING', str(istart)),
             bq.ScalarQueryParameter('end', 'STRING', str(iend)),
-            bq.ScalarQueryParameter('project', 'STRING', utils.SEQR_PROJECT_ID),
+            bq.ScalarQueryParameter('projects', 'STRING', projects),
```
Comment (illusional) [inline cpg_infra/billing_aggregator/aggregate/seqr.py:328]: I think this is supposed to be a `bq.ArrayQueryParameter`

**Review by illusional — CHANGES_REQUESTED** [review]
Can you double check that ScalarQueryParameter is correct for the list of projects?

File: cpg_infra/billing_aggregator/aggregate/seqr.py (hunk 1 above)
Comment (violetbrina) [reply cpg_infra/billing_aggregator/aggregate/seqr.py:315]: Will do.

File: cpg_infra/billing_aggregator/aggregate/seqr.py (hunk 2 above)
Comment (violetbrina) [reply cpg_infra/billing_aggregator/aggregate/seqr.py:328]: Ah of course. Good catch.

**Comment by violetbrina** [comment]
Run locally, all works!

File: cpg_infra/billing_aggregator/aggregate/gcp.py [hunk 3]
```diff
@@ -128,15 +128,14 @@ def get_billing_data(start: datetime, end: datetime):
         FROM `{utils.GCP_BILLING_BQ_TABLE}`
         WHERE export_time >= @start
             AND export_time <= @end
-            AND project.id <> @seqr_project_id
+            AND project.id NOT IN UNNEST(@projects)
```
Comment (illusional) [inline cpg_infra/billing_aggregator/aggregate/gcp.py:131]: `projects` isn't defined in the ArrayQueryParameter below, I'm also a little surprised you need to UNNEST projects.

**Pull request closed without merging**